src/components/PostsList.tsx

```diff
@@ -1,86 +1,66 @@
 import React from 'react';
+import classNames from 'classnames';
+import { Post } from '../types/Post';
 
-export const PostsList: React.FC = () => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
+type Props = {
+  posts: Post[];
+  setIsForm: (value: boolean) => void;
+  selectedPost: Post | null;
+  setSelectedPost: (value: Post | null) => void;
+};
 
-    <table className="table is-fullwidth is-striped is-hoverable is-narrow">
-      <thead>
-        <tr className="has-background-link-light">
-          <th>#</th>
-          <th>Title</th>
-          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-          <th> </th>
-        </tr>
-      </thead>
+export const PostsList: React.FC<Props> = ({
+  posts,
+  setIsForm,
+  selectedPost,
+  setSelectedPost,
+}) => {
+  const handleSelectedPost = (post: Post) => {
+    if (selectedPost?.id !== post.id) {
+      setSelectedPost(post);
+      setIsForm(false);
+    } else {
+      setSelectedPost(null);
+    }
+  };
 
-      <tbody>
-        <tr data-cy="Post">
-          <td data-cy="PostId">17</td>
+  return (
+    <div data-cy="PostsList">
+      <p className="title">Posts:</p>
 
-          <td data-cy="PostTitle">
-            fugit voluptas sed molestias voluptatem provident
-          </td>
+      <table className="table is-fullwidth is-striped is-hoverable is-narrow">
+        <thead>
+          <tr className="has-background-link-light">
+            <th>#</th>
+            <th>Title</th>
+            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+            <th> </th>
+          </tr>
+        </thead>
 
-          <td className="has-text-right is-vcentered">
-            <button
-              type="button"
-              data-cy="PostButton"
-              className="button is-link is-light"
-            >
-              Open
-            </button>
-          </td>
-        </tr>
+        <tbody>
+          {posts.map((el) => (
```

Comment:
Try to write more meaningful variable naming 
```suggestion
          {posts.map((post) => (
```
OR
```suggestion
          {posts.map((postItem) => (
```